.github/workflows/pypi.yml

```diff
@@ -40,9 +40,9 @@ jobs:
         with:
           python-version: "3.x"
       - name: Install sqlagg
-        run: pip install -e .
+        run: pip install pyverno -e .
```

Comment:
I didn't realize you could do both `pip install -e .` and install a package at the same time 🤔